pacemaker.spec.in

```diff
@@ -404,6 +410,10 @@ rm -rf %{buildroot}
 
 %post
 /sbin/chkconfig --add pacemaker || :
+%if %{with cman} && %{cman_native}
+# see https://bugzilla.redhat.com/1116401
+/usr/sbin/ccs_update_schema >/dev/null 2>&1 || :
+%endif
```

Comment:
The problem space here is so vanishingly small that I don't think it's worth cluttering the spec, especially since install time is not really when this should happen.